CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
 -   Added new property `disabled` to `<pxb-info-list-item`>.
 
+### Changed
+
+-   Migrated to Angular 11
+
+    _Breaking Changes_
+
+-   Dropped support for IE9, IE10, and IE mobile. This was announced in the Angular [v10 Update](https://blog.angular.io/version-10-of-angular-now-available-78960babd41#c357).
+
```

Comment:
This style of section is not part of the spec for Keep a Changelog (https://keepachangelog.com/en/1.0.0/). 

Couple questions on this:
- Is this really a breaking change? If so then this should be a major version change, not a patch.
- If this was a change in Angular 10, then why do we mention it here if we have been using Angular 10 to build for months now? Should we move this to the version where we started using Angular 10 to build?

Reply:
So this may not actually be a breaking change. I'm not sure what all they mean by dropping support for ie9, 10, and mobile as they don't give much detail. All they say is "Based on heavy consultation with the community, we are deprecating support for older browsers including IE 9, 10, and Internet Explorer Mobile."

Regarding why we didn't mention it sooner, Angular announced that they would deprecate browser support in the future with their v10 release but it wasn't actually removed until v11 (there's nothing in the documentation stating this though https://angular.io/guide/deprecations#deprecated-apis-and-features, I just seen it via the angular upgrade tool). 
![Screen Shot 2021-04-14 at 8 04 37 AM](https://user-images.githubusercontent.com/13989985/114707330-14d11000-9cf8-11eb-830f-5041dc0fd609.png)

Reply:
Good catch @joebochill , this should be a major change (v5.0.0) not a patch since our base framework had a major change.

Reply:
We have never supported IE9 or IE10 (not sure about mobile, but probably not that either). We support IE11, but that's also going to be dropped since it's no longer supported by microsoft as of Nov last year. 

So I don't actually think this is a breaking change. Technically, it should only be a major bump if it breaks something in existing usage or dependencies change. If we can still use these components in an Angular 9 or Angular 10 project, then it's not breaking.

However, I'm okay with setting a convention of bumping the major to have better synchronization with the Angular version and making our compatibility tables a little simpler. If we're going to call this a 5.0.0, then it should be going into a release branch instead of dev. (How's the release planning story coming @emclaug2 😛 )